scania-digital-design-system/sdds
# feat(utility-classes): add utility classes to storybook

**Describe pull-request**  
Adds the utility-classes in storybook.

**Solving issue**  
Fixes: [AB#2594](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2594)

**How to test**  
1. Go to storybook link below.
2. Check in Utilities -> Utility classes
3. Check all stories.

**Screenshots**  
-

**Additional context**  
-
